tests/lights/test_light.py

```diff
@@ -298,9 +347,33 @@ async def test_enum_light_properties_no_key(deconz_light):
     """Verify enum properties return expected values or None."""
     light = await deconz_light({"config": {}, "state": {}, "type": "Color light"})
 
+    assert light.supported_effects is None
     assert light.alert is None
     assert light.color_capabilities is None
     assert light.color_mode is None
     assert light.effect is None
     with pytest.raises(KeyError):
         assert light.fan_speed
+
+
+async def test_enum_light_properties_no_capabilities_color_key(deconz_light):
```

Comment:
Combine the tests to one using pytest parametrize to modify the expected payload.